Libplanet.Stun/Stun/TurnClient.cs

```diff
@@ -281,28 +282,32 @@ public async Task ReconnectTurn(int listenPort, CancellationToken cancellationTo
             return !_control.Client.LocalEndPoint.Equals(mapped);
         }
 
-        public async Task<bool> IsConnectable(
-            CancellationToken cancellationToken = default)
+        public async Task<bool> IsConnectable(int probeLength = ConnectableProbeLength)
         {
+            CancellationTokenSource cts = new CancellationTokenSource();
             try
             {
-                using var client = new TcpClient();
+                using TcpClient client = new TcpClient();
```

Comment:
I think that `var` is fine in this case because type is clear due to`new TcpClient()` 😅